src/fal/cli/__init__.py

```diff
@@ -46,3 +58,33 @@ def cli(argv=sys.argv):
                 exclude_count=exclude_count,
                 script_count=script_count,
             )
+
+
+def _before_fal_scripts(node_graph: NodeGraph, plan: ExecutionPlan) -> List[FalScript]:
```

Comment:
Should we change the method to get `List[str]` instead of `ExecutionPlan` and the caller either passes `plan.before_scripts` or `plan.after_scripts`

Reply:
yup makes sense